celery_utils/chordable_django_backend.py

```diff
@@ -0,0 +1,219 @@
+u"""
+An opt-in backend, designed to make using chords with the Django ORM easy.
+
+Additionally avoids the pooling unlock used by default in django-celery for
+performance benfits.
+
+Usage
+-----
+    First, import the needful from this module:
+        from celeryutils.chordable_django_backend import chord, chord_task
+
+    Then, change the task() decorators on your functions to the new chord_
+    decorators:
+        # Note that standard task options will be passed though!
+        @chord_task(max_retries=2, default_retry_delay=10)
+        def add(x, y):
+            return x + y
+
+        @chord_task()
+        def tsum(results):
+            _sum = 0
+            for num in results:
+                _sum = _sum + num
+            return _sum
+
+    Finally, call chord() as you normally would, and let ChordableDjangoBackend
+    take care of the rest:
+        chord(add.s(i, i) for i in xrange(10))(tsum.s())
+
+Notes
+-----
+    Please ensure that your callback function stores its results somewhere.
+    There is no "chord finished, come and get it!" signal, so having the
+    callback store the synthesized results of your parallel subtasks is
+    crucial.
+
+    By default, errors in the subtasks will prevent the callback from running.
+    Instead, the callback result will be set to FAILURE status and the result
+    will contain an exception detailing which subtasks errored. The other option
+    is <TODO FIGURE OUT THE DEV NOTE IN MODELS.PY AND PUT RESULTS HERE>, which
+    can be set with a False use_iterator option on your callback signature.
+
+    Similarly, the callback is, by default, called with an iterator of subtask
+    results. Since this cannot be serialized, it cannot be launched
+    asynchronously, and the final subtask will execute the callback in-process.
+    In order to disable this behavior (and build the results list in-memory),
+    set the propagate option on your callback signature to False.
+
+    To set either of the use_iterator or propagate options, add them to your
+    callback signature as follows:
+        c = chord(add.s(i, i) for i in xrange(10))
+        t = tsum.s()
+        t.options = {
+            'propagate': False,
+            'use_iterator': False
+        }
+        c(t)
+
+"""
+from __future__ import absolute_import
```

Comment:
Consider using the full four-item __future__ import to make code behavior more predictable across the repo, and more consistent with python 3.

    from __future__ import absolute_import, division, print_function, unicode_literals